src/Tokens.ts

```diff
@@ -0,0 +1,12 @@
+// Copyright (c) ZALORA
```

Comment:
I'm not sure if we've yet defined a code guideline around this but i believe that for files that are exporting an object with the same name it should be PascalCased, and for files that export many things and none have the name of the file it should be lowercased. example:

```
import WrapperThing from 'WrapperThing';
import { add, subtract, mul, div } from 'calculator';
``` 

So how about we change the name of this file to tokens.ts? (btw the easiest way to do this is git mv: `git mv src/Tokens.ts src/tokens.ts`)